lnrpc/walletrpc/walletkit_server.go

```diff
@@ -516,7 +516,13 @@ func (w *WalletKit) BumpFee(ctx context.Context,
 		HashType: txscript.SigHashAll,
 	}
 
-	input := input.NewBaseInput(op, witnessType, signDesc, 0)
+	_, currentHeight, err := w.cfg.Chain.GetBestBlock()
+	if err != nil {
+		return nil, fmt.Errorf("unable to retrieve current height: %v",
+			err)
+	}
+
+	input := input.NewBaseInput(op, witnessType, signDesc, uint32(currentHeight))
```

Comment:
I think usually the height hint that we pass to the sweeper is the height at which the input was created. So that in case the input is swept by something else (maybe an old tx of ourselves or the remote party), we don't miss the spend ntfn. For an external input, I think we need to pass that as a parameter with the rpc call?

Reply:
The output is unconfirmed so we can assume the current height as the height hint. Only thing is that this RPC currently doesn't validate the output is indeed unconfirmed.

Reply:
Ah yes, that is right. And what will happen if you `bumpfee` a confirmed output? (suppose it confirms just when you are typing the command). The sweeper won't detect the spend and try to sweep `maxAttempts` times? Does it need some padding here, like `currentHeight-x`?

Reply:
If you'd want to add a check for confirmedness in the rpc, you'd need the height hint right? But if you have a proper height hint, you can just as well pass that into the sweeper. The sweeper will then get the spend ntfn immediately and report the outcome. So no checking on the rpc level would be required? Just thinking about your comment, not saying that this needs to be added now.

Reply:
Ok, I am mixing up conf and spent. In any case, if we want to do checking, we may want to do it in `SweepInput` and not on the rpc level.

Reply:
> And what will happen if you bumpfee a confirmed output? 

You're essentially just sending to yourself at that point for no gain.

I think all that's needed is a check at the RPC level to detect if the output is unconfirmed. If it's confirmed then we can return an error.

Reply:
Unconfirmed check added.